pennylane/ops/qutrit/matrix_ops.py

```diff
@@ -98,20 +109,232 @@ def compute_matrix(U):  # pylint: disable=arguments-differ
 
         Returns:
             tensor_like: canonical matrix
+
+        **Example**
+
+        >>> U = np.array([[1, 1, 0], [1, -1, 0], [0, 0, np.sqrt(2)]]) / np.sqrt(2)
+        >>> qml.QutritUnitary.compute_matrix(U)
+        array([[ 0.70710678,  0.70710678,  0.        ],
+               [ 0.70710678, -0.70710678,  0.        ],
+               [ 0.        ,  0.        ,  1.        ]])
         """
         return U
 
     def adjoint(self):
         U = self.matrix()
         return QutritUnitary(qml.math.conj(qml.math.moveaxis(U, -2, -1)), wires=self.wires)
 
-    # TODO: Add `_controlled()` method once `ControlledQutritUnitary` is implemented.
     # TODO: Add compute_decomposition() once parametrized operations are added.
 
     def pow(self, z):
         if isinstance(z, int):
             return [QutritUnitary(qml.math.linalg.matrix_power(self.matrix(), z), wires=self.wires)]
         return super().pow(z)
 
+    def _controlled(self, wire):
+        new_op = ControlledQutritUnitary(*self.parameters, control_wires=wire, wires=self.wires)
+        return new_op.inv() if self.inverse else new_op
+
     def label(self, decimals=None, base_label=None, cache=None):
         return super().label(decimals=decimals, base_label=base_label or "U", cache=cache)
+
+
+class ControlledQutritUnitary(QutritUnitary):
+    r"""ControlledQutritUnitary(U, control_wires, wires, control_values)
+    Apply an arbitrary fixed unitary to ``wires`` with control from the ``control_wires``.
+
+    In addition to default ``Operation`` instance attributes, the following are
+    available for ``ControlledQutritUnitary``:
```

Comment:
This sentence makes it sound like `U` and `control_values` are "attributes", and therefore accessible like `op.U` and `op.control_values`. Should we make them properties? 

Or we can just remove this blurb. The available attributes are accessible under "Attributes" in the documentation anyway.

Reply:
I see your point. I do see the benefit of highlighting which attributes/properties are available to users, but as you said, it is a bit redundant. I think that making `control_values` a property is helpful definitely for the sake of convenient access, but I'm not entirely sure if accessing `op.U` makes much sense as in most use cases users will likely already have access to the unitary. I've added `control_values` as a property for now. Let me know what you think!

Reply:
Thanks for adding the property.  I added a suggestion to clarify that `op.U` itself isn't an attribute, but accessible via `op.parameters[0]`.